### ISSUE TITLE
LogManager error of type WRITE_FAILURE

### ISSUE THREAD

**Issue by MarcusBiel** [quarkiverse/quarkus-logging-logback#13]
I have quarkus 2.1.1 and 

        <dependency>
            <groupId>io.quarkiverse.logging.logback</groupId>
            <artifactId>quarkus-logging-logback</artifactId>
            <version>0.2.0</version>
        </dependency>
        <dependency>
            <groupId>org.jboss.logging</groupId>
            <artifactId>commons-logging-jboss-logging</artifactId>
        </dependency>
        <dependency>
            <groupId>ch.qos.logback</groupId>
            <artifactId>logback-classic</artifactId>
            <version>1.2.5</version>
        </dependency>
        <dependency>
            <groupId>org.slf4j</groupId>
            <artifactId>slf4j-api</artifactId>
            <version>1.7.32</version>
        </dependency>

When I locally start Quarkus with my Application.jar from my IDE, it startups fine,

but when instead I try:
java -jar ./target/quarkus-app/quarkus-run.jar
LogManager error of type WRITE_FAILURE: The delayed handler's queue was overrun and log record(s) were lost. Did you forget to configure logging?

I assumed it could be related also to the environment specific application.properties set up I have -
In application.properties, I have added all config entries that are equal to all environments.
In application-dev.properties, I have added all config centries for the dev profile.

To ensure I am running on the dev profile, I had also added -Dquarkus.profile=dev, also I tried -DQUARKUS_PROFILE=dev, but the above error keeps re-appering. No idea why.

**Comment by stuartwdouglas** [comment]
Do you have lots of TRACE or DEBUG logging at startup? If log messages are generated before logging is configured they are buffered, but there is a limit to how many are buffered. If the limit (default 4k) is hit some of them are dropped (lowest priority first). You can use -Dquarkus-log-max-startup-records=100000 to change the limit, but it seems odd this many messages are generated before logging is setup.

**Comment by MarcusBiel** [comment]
No, I never use TRACE or DEBUG at all, and I generally limit logging to a minimum.

**Comment by stuartwdouglas** [comment]
Something would be generating those messages though, and they are buffered until logging is configured (as we don't know if they should be logged until we have actually read all the config). 

Can you set `-Dquarkus-log-max-startup-records=100000` and set logging to TRACE level, and see what is generating all the messages?
